main/src/ca/uwaterloo/flix/language/ast/ExecutableAst.scala

```diff
@@ -200,6 +200,8 @@ object ExecutableAst {
 
     case class Var(sym: Symbol.VarSym, tpe: Type, loc: SourceLocation) extends ExecutableAst.Expression
 
+    case class Statement(exp1: ExecutableAst.Expression, exp2: ExecutableAst.Expression, tpe: Type, loc: SourceLocation) extends ExecutableAst.Expression
```

Comment:
I think the Statement case class should follow the IfThenElse case class since these two deal with control flow.